reviews/models/mixins.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from transitions import Machine
+
+from django.db import models
+
+from reviews import workflow
+
+
+class ReviewProviderMixin(models.Model):
```

Comment:
It may be nice to wrap up all review related columns into an object and expose that as `object.review[_settings]`. `object.review_comments_public` -> `object.review[_settings].comments_public`.

It's a bit opaque but could provide a slightly nicer interface.

A quick google search didn't turn up an libraries that automatically do this for you, but I've seen [some ORMs](https://stackoverflow.com/questions/2669125/can-a-django-model-have-embed-classes) in [other languages](https://typeorm.github.io/tables-and-columns.html#embeddables) do something similar.